integration-cli/docker_cli_daemon_test.go

```diff
@@ -2070,3 +2070,20 @@ func (s *DockerDaemonSuite) TestRunLinksChanged(c *check.C) {
 	c.Assert(err, check.NotNil, check.Commentf(out))
 	c.Assert(out, check.Not(checker.Contains), "1 packets transmitted, 1 packets received")
 }
+
+func (s *DockerDaemonSuite) TestDaemonUngracefulShutdownCleanUpRootfsOfOverlay(c *check.C) {
```

Comment:
We have a more generic test about cleaning up: `TestCleanupMountsAfterCrash`. Do you think it might be enough? Or what we testing here?